src/SYNFPAN.m

```diff
@@ -366,6 +366,14 @@ s MISCARY("LAB_TEST",VLAB)=value
  N MAP
  ; vistapanel
  S MAP="vistapanel"
+ ; CBC W Differential panel, method unspecified - Blood (69742-5) -> CBC
+ S @LOC@(MAP,"CODE","69742-5","CBC")=""
+ ; Auto Differential panel - Blood (57023-4) -> DIFFERENTIAL COUNT
+ S @LOC@(MAP,"CODE","57023-4","DIFFERENTIAL COUNT")=""
+ ; Gas panel - Arterial blood (24336-0) -> GAS PANEL - ARTERIAL
+ S @LOC@(MAP,"CODE","24366-0","GAS PANEL - ARTERIAL")=""
+ ; Gas panel - Venous blood (24339-4) -> BLOOD GASES
+ S @LOC@(MAP,"CODE","24339-4","BLOOD GASES")=""
```

Comment:
This was already added by me (sorry) and you are now repeating it. But you still need to QA the results.

Reply:
I deleted the duplicate I added